x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/action/saml/SamlSpMetadataAction.java

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+package org.elasticsearch.xpack.core.security.action.saml;
+
+import org.elasticsearch.action.ActionType;
+
+public class SamlSpMetadataAction extends ActionType<SamlSpMetadataResponse> {
+    public static final String NAME = "cluster:admin/xpack/security/saml/metadata";
```

Comment:
Is it intentional that this is `admin` rather `monitor`?

Reply:
Good catch. I think that this could be monitor as it fits under what I can envision a user with a role with `monitor` privilege could do without needing `manage_saml` or `manage_security`

Reply:
Good catch!

Reply:
Fixed